### ISSUE TITLE
Update Sentry docs once #1411 is merged.

### ISSUE THREAD

**Issue by lucas-zimerman** [getsentry/sentry-dotnet#1413]
replace 
```csharp
    protected void Application_Error()
    {
        var exception = Server.GetLastError();
        SentrySdk.CaptureException(exception);
    }
``` 
by

```csharp
    protected void Application_Error() => Server.CaptureLastError();
```

**Comment by bruno-garcia** [comment]
It's released on 3.13.0

**Comment by SimonCropp** [comment]
docs PR https://github.com/getsentry/sentry-docs/pull/4703
